### ISSUE TITLE
subsetting rows that belong to a group

### ISSUE THREAD

**Issue by Jimmerkjaer** [SebKrantz/collapse#496]
Since updateing to R snapshot 2023-11, the possibility to subset rows in a grouped data.frame has been removed with the standard fsubset functionality.
I suppose it is due to the following line :
`fsubset.grouped_df <- function(.x, subset, ...) stop("fsubset() does not support grouped data: please subset your data before grouping it")`

As a hack one could just specify the `fsubset.data.frame()` but I wonder why it is no longer allowed to use the fsubset() function for grouped data.frames ?

As an example I hope you would be so kind as to provide a collapse-version of the following dplyr-approach:

`df <- df%>% group_by(x, y, z) %>% filter(n() > 1) %>% ungroup() %>% distinct() %>% as.data.frame()`

as this could previously be done with

`df <- df %>% fgroup_by(x, y, z) %>% fsubset(GRPN(.) > 1) %>% fungroup() %>% funique()`

**Comment by Jimmerkjaer** [comment]
I have discovered that the following might be a solution to the mentioned problem:
`df <- df %>% fsubset(GRPN(list(x, y, z)) > 1) %>% funique()`
Is this recommendable or is there another syntax that is preferred ?

**Comment by SebKrantz** [comment]
Hi, thanks. The truth is that `fsubset()` never did anything on grouped data. It had no method for that, only for data frame. It is only after I realized that people are doing this that I decided to create a .grouped_df method with an error. Please read the following section of the *collapse* for tidyverse users vignette: https://sebkrantz.github.io/collapse/articles/collapse_for_tidyverse_users.html#writing-efficient-code

In summary: in efficient data manipulation code it is never necessary to subset grouped data, the grouping step should always come after subsetting.
